client/src/pages/Landing/LandingForm/LandingForm.tsx

```diff
@@ -0,0 +1,110 @@
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import CircularProgress from '@mui/material/CircularProgress';
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
+import useStyles from './useStyles';
+import FormInput from '../../../components/FormInput/FormInput';
+import FormInputRight from '../../../components/FormInput/FormInputRight';
+import FormInputLeft from '../../../components/FormInput/FormInputLeft';
+
+interface Props {
+  handleSubmit: (
+    {
+      location,
+      start,
+      end,
+    }: {
+      location: string;
```

Comment:
We can use an interfaces where applicable to minimize retyping the params and types